transports/gemini-live-websocket-transport/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pipecat-ai/gemini-live-websocket-transport",
-  "version": "0.3.4",
+  "version": "0.3.5",
```

Comment:
Should we go ahead and bump this directly to `0.3.6` to keep all the transports in sync ? What do you think ?

Reply:
Actually, never mind, there is probably no reason for all of them to have the same version.